docs/rule/no-autofocus-attribute.md

```diff
@@ -0,0 +1,30 @@
+# no-autofocus-attribute
+
+🔧 The `--fix` option on the command line can automatically fix some of the problems reported by this rule.
+
+Enforce no autofocus prop on element. The autofocus attribute is a global attribute that indicates an element should be focused on page load. Because autofocus reduces accessibility by moving users to an element without warning and context, it should not be used.
+
+This rule takes no arguments.
+
+## Examples
+
+This rule **allows** the following:
+
+```hbs
+<input />
+```
+
+This rule **forbids** the following:
+
+```hbs
+<input autofocus />
+```
+
+```hbs
+<input autofocus='true' />
```

Comment:
Why not simply `<input autofocus>`? (I know the PR was merged)

Reply:
Because `autofocus="autofocus"` is also a valid way of defining the attribute and we also show `autofocus` in the docs as well:
https://github.com/ember-template-lint/ember-template-lint/pull/2168/files#diff-8ce4ad24f0b3c9100092b7fa447f7f763c1867cdd986320c798b0742ff28d275R16

Reply:
Oh dang... I am sorry, I did not see it was also shown on the document a few lines above 😑 .

Reply:
No worries 😃